Design Repo Restructure

Shrinks99 opened the pull request:
## A full restructure of how assets are stored and managed!

This retains the vast majority of the content that the artworks repo contained, I removed two working assets (meshnet-logos.xcf and nodes.svg) and if anyone misses those I will re-add them.  I doubt anyone will though... :)

- Files are now organized by a project folder.  Graphics that are currently being used go inside the 'Current Assets' directory.  Depreciated graphics go into 'Legacy Assets'

- Citations and licensing for graphics that are used under a license are to be listed in 'Asset Usage Licenses.md'

- There is a new README.md file that reflects these changes

- This branch also includes my updated ToMesh City Hall sticker.  It's not related to the restructure but sorta just got lumped in there ¯\\\_(ツ)_/¯

Shrinks99 committed:
Folder and asset restructure

Things are now grouped by project, deleted the XCF file for the mesh circle diagram and the SVG three hexagon template as they are very old and will no longer used for anything ever.

Shrinks99 committed:
Created asset usage licenses

This file tracks the licensing and credits for graphics used under license / with credit.  Adapted from the previous readme.md

Shrinks99 committed:
New readme file

The start of a new README.md

Existing information has been placed in the 'Asset Usage Licenses.md' file.

Shrinks99 committed:
Created master "legacy assets" and "current assets" folders

These differentiate between outdated and current content by project.  Updated the readme to reflect these changes.

dcwalk commented on README.md:
```diff
@@ -1,36 +1,11 @@
-  - Relies on GTA boundary map by [Mortadelo2005](https://en.wikipedia.org/wiki/File:Greater_toronto_area_map.svg) CC BY-SA 3.0
+There are no explicit restrictions of graphics file types although generally open formats are favourable to proprietary ones.  Toronto Mesh website code should be directed towards the [tomesh.net](https://github.com/tomeshnet/tomesh.net) repository.
 
-<span xmlns:dct="http://purl.org/dc/terms/" property="dct:title">Toronto Mesh sticker designs</span> are licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
```
The license block should stay in the readme, perhaps adding a license file

Shrinks99 replied:
Should it stay in the readme?  I added a licenses file to keep them separate from the readme so that it didn't become real long in the future but if we want to keep them there I guess that's cool and it can just be its own section

dcwalk commented:
Awesome that you are tackling this!

I'm not entirely clear the current vs. legacy distinction, perhaps in part because it combines logos designed with/for tomesh and those that werent. My sense is that is more important to keep as a distinction?

A couple other thoughts:

- The "Our Networks" stickers were printed for the event, but only the "soon" one is actually a design for the event. The rest were general tomesh logos (i.e., wave and meshnet) and voted on as most popular, so I think they should be grouped at sticker designs, but if y'all are deprecating them that is totally fine
- I would favour not cutting vector and open editable formats 
- In general we avoid spaces in folder names, it would be my strong preference that we keep that practice up as that conforms to popular usage elsewhere (i.e., see [discussion here](https://unix.stackexchange.com/questions/239295/is-it-considered-a-best-practice-to-not-use-capital-letters-in-file-naming))
- similarly I think we often avoid first letter capitalization. I think there are different, less strong feelings about this but want to admit it is my preference

dasanchez reviewed (changes requested):
@Shrinks99 this looks great!
- Formatting: I also prefer to see files and folders names in lowercase, separated by a dash.
- Can you add a svg or png file for the wifi logo? 
Thanks for doing this!

dcwalk commented:
Also, just realized there are two repos? Can we remove the one that is no longer being used? (It looks like the commit log is preserved here and I assume the old one will be deprecated)

darkdrgn2k commented:
We had to recreate the repo to remove the fork.

Once we all agree on things we will remove the old one.

#2 motion to delete artwork repo in favour of this one

Shrinks99 commented:
These changes are great and will be made!  The file naming is my personal preference for my file system but best to stay consistent with ours (slipped my mind :P ).

The sticker and logo designs were before my time so I don't have the best context as to how they should be arranged.  That said, they are all unused currently and IMO fit into a "graphics we don't currently use but should keep because we shouldn't delete our things we've made" folder, that's the idea behind 'legacy assets'.  'current assets' is for graphics that should currently be used and have not been replaced / depreciated, separating graphics into these two categories makes it easy to find whatever up to date graphic people may need in the future.

TL;DR if we want to rename / reorganize the folders in there that's cool with me but all of those graphics should stay in 'legacy graphics'.

Shrinks99 committed:
fixes file naming scheme, adds a few things to the readme

Adds standards for file types and suggested inclusions for open formats.  File names are now in line with the Toronto Mesh naming scheme.

Shrinks99 committed:
readme updates

Improved wording

dcwalk commented on asset-usage-licenses.md:
```diff
@@ -0,0 +1,21 @@
+- `tomesh-sticker-soon` [dcwalk](https://github.com/dcwalk)
+  - Relies on GTA boundary map by [Mortadelo2005](https://en.wikipedia.org/wiki/File:Greater_toronto_area_map.svg) CC BY-SA 3.0
+
+<span xmlns:dct="http://purl.org/dc/terms/" property="dct:title">Toronto Mesh sticker designs</span> are licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
```
Could we have this repo follow the pattern of including licensing information in the README? That plus an included `LICENSE` is a popular practice and I think we should have a really good reason if we want to deviate from it.
I'm not sure that this separate file will be clearer, esp. if not linked anywhere.

Shrinks99 replied:
I just didn't want the readme to get too full of stuff, especially because
licensing graphics will most likely be more prevalent than licensing in
other repos.  We can include it in the readme or include a link to the
separate license file.  We could also have both although that would mean there
would be two different files for the same purpose and that seems like a bad plan.

Up to you!  I'm willing to go with the flow here.

dasanchez replied:
Let's include the LICENSE file at the top level of the repo- here's a copy:
https://github.com/jslicense/CC-BY-SA-4.0/blob/master/CC-BY-SA-4.0

A note in the README referencing the license is enough for me.

For reference:
https://help.github.com/articles/licensing-a-repository/#determining-the-location-of-your-license

Shrinks99 commented:
I just didn't want the readme to get too full of stuff, especially because
licensing graphics will most likely be more prevalent than licensing in
other repos.  We can include it in the readme or include a link to the
separate readme.  We could also have both although that would mean there
would be two different files for the same purpose.

Up to you!  I'm willing to go with the flow here.

On Sun, May 13, 2018 at 1:59 PM dcwalk <notifications@github.com> wrote:

> *@dcwalk* commented on this pull request.
> ------------------------------
>
> In asset-usage-licenses.md
> <https://github.com/tomeshnet/design/pull/1#discussion_r187808843>:
>
> > @@ -0,0 +1,21 @@
> +# Asset Credits and License Requirements
> +
> +
> +## Our Networks 2017 Stickers
> +
> +Credits for the graphics under 'Our Networks 2017 Stickers'
> +
> +- `tomesh-sticker-meshnet` [WilliamRutherford](https://github.com/WilliamRutherford)
> +- `tomesh-sticker-waves` [Hamalittle](https://github.com/Hamalittle)
> +- `tomesh-sticker-soon` [dcwalk](https://github.com/dcwalk)
> +  - Relies on GTA boundary map by [Mortadelo2005](https://en.wikipedia.org/wiki/File:Greater_toronto_area_map.svg) CC BY-SA 3.0
> +
> +<span xmlns:dct="http://purl.org/dc/terms/" property="dct:title">Toronto Mesh sticker designs</span> are licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/">Creative Commons Attribution-ShareAlike 4.0 International License</a>.
>
> Could we have this repo follow the pattern of including licensing
> information in the README? That plus an included LICENSE is a popular
> practice and I think we should have a really good reason if we want to
> deviate from it.
> I'm not sure that this separate file will be clearer, esp. if not linked
> anywhere.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/tomeshnet/design/pull/1#pullrequestreview-119662133>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFaPatsHMsoRva6uFFP2W4vHokyGH0BEks5tyHR9gaJpZM4T8qhA>
> .
>
-- 
-Hank

dasanchez committed:
Added LICENSE

- CC-BY-SA-4.0 license file added to root folder
- License and credits sections added to README

dasanchez committed:
Removed asset-usage-licenses.md

Moved Meshnet Community Assets section to README file.

darkdrgn2k reviewed (approved):
Thanks for doing this!
